product_docs/docs/mongo_data_adapter/5.3.0/02_requirements_overview.mdx

```diff
@@ -12,12 +12,18 @@ The MongoDB Foreign Data Wrapper is supported on the following platforms:
 
 **Linux x86-64**
 
- -   RHEL 8/Rocky Linux 8/AlmaLinux 8/OL 8
- -   RHEL 7/CentOS 7/OL 7
+ -   RHEL 8
+ -   RHEL 7
+ -   Rocky Linux 8/AlmaLinux 8/OL 8
+ -   CentOS 7/OL 7
```

Comment:
OL 7

Reply:
You meant I should remove OL 7 from line 18, correct?